FastSimulation/ParticlePropagator/plugins/MagneticFieldMapESProducer.cc

```diff
@@ -1,30 +1,21 @@
 #include "FastSimulation/ParticlePropagator/plugins/MagneticFieldMapESProducer.h"
-#include "FastSimulation/TrackerSetup/interface/TrackerInteractionGeometryRecord.h"
-#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
-
 #include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
 #include "FWCore/Framework/interface/ModuleFactory.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 
-#include <memory>
```

Comment:
I'm sorry Sunanda, but the unused header I was talking about was "FWCore/Framework/interface/ESHandle.h" (now at line 2), and not this "memory" (which is used instead by the `unique_ptr`

Reply:
I can remive the ESHandle.h. Also <memory> isdeclared in the .h file. So I can keep one of them.

Reply:
> I can remive the ESHandle.h. Also isdeclared in the .h file. So I can keep one of them.

Ok, thank you for telling that the header includes where already in the .h file.
If so, there is no need to remove ESHandle: if it is the only header that remain to be fixed, that is a really minor thing.